cpanfile

```diff
@@ -52,6 +53,10 @@ requires 'Test::Synopsis';
 requires 'Test::Version';
 requires 'Test::Warnings';
 
+if ( "$]" >= 5.008001 ) {
```

Comment:
we are building `5.8` but wonder what version of 5.8 we are providing, I bet this check is useless?

Reply:
Deleted when I realised the 5.8 image was 5.8.9 (and there was no 5.6 image).